viz/metrics-api/stat_summary.go

```diff
@@ -55,7 +55,8 @@ const (
 	httpAuthzAllowQuery  = "sum(increase(inbound_http_authz_allow_total%s[%s])) by (%s)"
 	tcpConnectionsQuery  = "sum(tcp_open_connections%s) by (%s)"
 	tcpReadBytesQuery    = "sum(increase(tcp_read_bytes_total%s[%s])) by (%s)"
-	tcpWriteBytesQuery   = "sum(increase(tcp_write_bytes_total%s[%s])) by (%s)"
+	// This is not a hardcoded credential
```

Comment:
Yep I wasn't sure why it hit this variable and not the others. I think it was because of `Write` showing up in the variable name. For example if you change the variable name to not include `Write` or `Wrt` then the ignore isn't needed.

I defaulted to including the the ignore line rather than renaming the variable to something like `tcpWBytesQuery` though.

Reply:
It's probably from the `pattern` used [here](https://github.com/securego/gosec/blob/master/rules/hardcoded_credentials.go#L120). It may be seeing the `pw` part of the variable name and assigning it a threshold large enough to issue a warning.

Reply:
heh, that's hilarious.  given that this rule has a number of false positives across our code, perhaps it's worth just disabling this rule?

Reply:
Yep will disable.